x-pack/metricbeat/module/sql/query/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: query
+  type: group
+  release: beta
+  description: >
+    query
+  fields:
+    - name: example
+      type: keyword
+      description: >
+        Example field
```

Comment:
This example field can be removed.

Reply:
Done.

Reply:
I meant the whole field definition :slightly_smiling_face: